oauth-common/src/main/java/io/strimzi/kafka/oauth/common/Config.java

```diff
@@ -0,0 +1,135 @@
+package io.strimzi.kafka.oauth.common;
+
+import java.util.Properties;
+
+public class Config {
```

Comment:
does it make sense to name it more specifically as `OAuthConfig` instead of just `Config`?

Reply:
It makes sense - to make it more IDE friendly (code completion).

Reply:
But I tried and I don't like it. It starts to stick out among other classes, and the code is then more verbose. Not worth it.

Reply:
Tbh I didn't get your point. If I read a class just named `Config` it's not simple to understand that it refers to OAuth without the entire package name.

Reply:
Try it, and you'll see. It sticks out from other classes like a sore thumb, and makes the code ugly. As soon as you see it you immediately start to wonder why not prefix other classes in that package as well. But then you're moving into class name a naming scheme that packages are supposed to solve.